package.json

```diff
@@ -31,7 +31,7 @@
     "threear": "github:mn-pollinators/THREEAR#ts-fixes-2",
     "tslib": "^1.10.0",
     "zone.js": "~0.10.2",
-    "@mn-pollinators/assets": "github:mn-pollinators/assets#npm-package"
+    "@mn-pollinators/assets": "0.2.0"
```

Comment:
Do we want to let the version float? (Something like `^0.2.0`?)

Reply:
If we did it would just be kept on place by `package.lock.json` anyway. I feel that we should be specific about the version of the assets we are using.